packages/cli/src/commands/spaces/transfer.ts

```diff
@@ -0,0 +1,29 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+
+export default class Transfer extends Command {
+  static topic = 'spaces';
+  static description = 'transfer a space to another team';
+  static help = 'Example:\n\n    $ heroku spaces:transfer --space=space-name --team=team-name\n    Transferring space-name to team-name... done\n';
```

Comment:
`static help` doesn't work in the OClif version. It would be better to move this example to the `static examples` property.